Content/default/paket.lock

```diff
@@ -381,8 +381,9 @@ NUGET
     Thoth.Json (5.1)
       Fable.Core (>= 3.1.6)
       FSharp.Core (>= 4.7.2)
-    TypeShape (9.0)
-      FSharp.Core (>= 4.3.2)
+    TypeShape (8.0.1)
```

Comment:
I would recommend avoiding taking a nuget dependency and embed the source file directly. You can then pin the version of the library you depend on at compile time.

https://github.com/eiriktsarpalis/TypeShape#installing